facades/PC/build.gradle.kts

```diff
@@ -200,12 +200,41 @@ tasks.register<JavaExec>("server") {
     args("-headless", "-homedir=$localServerDataPath")
 }
 
-// Preps a version file to bundle with PC dists. This eventually goes into the root of a zip file
-tasks.register<Copy>("createVersionFile") {
+
+/*********************************
+ * Distribution
+ *
+ * See also publish.gradle, included near the top.
+ */
+
+tasks.named<Jar>("jar") {
+    // Launcher expects the main class to be in the file with this name.
+    archiveFileName.set("Terasology.jar")
+
+    manifest {
+        //TODO: Maybe later add the engine's version number into here?
+        attributes["Main-Class"] = mainClassName
+        // A classpath in the manifest avoids the problem of having to put a classpath on the command line and
+        // "line is too long" errors: https://github.com/gradle/gradle/issues/1989
+        attributes["Class-Path"] = configurations["runtimeClasspath"].joinToString(" ") { it.name }
+        attributes["Implementation-Title"] = "Terasology-" + project.name
+        attributes["Implementation-Version"] = """${env["BUILD_NUMBER"]}, ${env["GIT_BRANCH"]}, ${env["BUILD_ID"]}"""
```

Comment:
https://github.com/MovingBlocks/Terasology/pull/4454#discussion_r569121967

GIT_BRANCH no longer used for this